lib/index.js

```diff
@@ -94,17 +135,23 @@ _applyPoints.forEach(function (applyPoint) {
 
             } else if (typeof routePolicy === 'function') {
 
+                if (!aggregateApplyPoint && routePolicy.runs && !routePolicy.applyPoint) {
+
+                    aggregateApplyPoint = determineAggregateApplyPoint(routePolicy.runs);
+                }
+
                 if (!hasValidApplyPoint(routePolicy)) {
```

Comment:
1. If the policy has an explicit `applyPoint`, that will be used.
2. Otherwise, an apply point will be determined from `run`.  In determining this, it requires that all listed policies were already placed on the same handler.
3. In the absence of `run` and `applyPoint`, the plugin's set `defaultApplyPoint` is used.